python/cuml/ensemble/randomforestregressor.pyx

```diff
@@ -741,3 +741,8 @@ class RandomForestRegressor(BaseRandomForestModel, RegressorMixin):
         if self.dtype == np.float64:
             return dump_rf_as_json(rf_forest64)
         return dump_rf_as_json(rf_forest)
+
+    def _more_tags(self):
+        return {
+            'preferred_input_order': 'F'
```

Comment:
F for fit, C for predict (awful, I know)... does that meet the tag definition?

Reply:
that's an excellent question, I guess for estimators that have discrepancies like this we should leave it as None, what do you think?